source/Calamari.Terraform/TerraformCliExecutor.cs

```diff
@@ -39,8 +38,9 @@ class TerraformCLIExecutor : IDisposable
             AllowPluginDownloads = variables.GetFlag(TerraformSpecialVariables.Action.Terraform.AllowPluginDownloads, true);
             AttachLogFile = variables.GetFlag(TerraformSpecialVariables.Action.Terraform.AttachLogFile);
             TerraformVariableFiles = GenerateVarFiles();
-
+            
             logPath = Path.Combine(deployment.CurrentDirectory, "terraform.log");
+            crashLogPath = Path.Combine(deployment.CurrentDirectory, "crash.log");
```

Comment:
It may make sense to add a link to https://www.terraform.io/docs/internals/debugging.html#interpreting-a-crash-log so we know where this `crash.log` comes from

Reply:
Done :)